src/appleseed/renderer/modeling/material/material.cpp

```diff
@@ -94,6 +94,20 @@ bool Material::has_alpha_map() const
     return get_non_empty(m_params, "alpha_map") != 0;
 }
 
+bool Material::has_uniform_opacity() const
```

Comment:
I liked the first suggestion, `is_uniformly_opaque()`: shorter, consistent with other methods, and IMHO descriptive enough... Oh well :)